index.html

```diff
@@ -107,6 +107,74 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long numFreezes;
+      unsigned long numPauses;
+      double totalFreezesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>numFreezes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video freezes experienced by this receiver. It is a
+          freeze if frame duration, which is time interval between consecutively
+          rendered frames, is 3 times higher than the average frame duration.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>numPauses</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video pauses experienced by this receiver. Video is
+          considered to be paused if time passed since last received packet
+          exceeds 5 seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>totalFreezesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total duration of video freezes, in seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>totalFramesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total duration of rendered video frames, in seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>sumOfSquaredFramesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned long">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Sum of squared duration of rendered frames. This can be used to
```

Comment:
What is "duration of rendered frames", when is it calculated?

Reply:
Updated definition:

"Sum of squared duration of all rendered frames. This value is updated when a frame is rendered. It can be used to calculate harmonic frame rate:
totalFramesDuration / sumOfSquaredFrameDurations"

That 'all' looks a bit redundant to me.

Reply:
> What is "duration of rendered frames", when is it calculated?

We have definition of frame duration in description of freezeCount. I'm not sure if adding a reference to that to each appearance of 'frame duration' would make sense.